src/controllers/update/update.sign.out.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Request, Response } from "express";
+import { createAndLogErrorRequest, logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    return res.render(config.UPDATE_SIGN_OUT_PAGE, {
+      previousPage: `${config.UPDATE_AN_OVERSEAS_ENTITY_URL}${req.query["page"]}`,
+      url: config.UPDATE_AN_OVERSEAS_ENTITY_URL
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const previousPage = req.body["previousPage"];
+    if (!previousPage.startsWith(config.UPDATE_AN_OVERSEAS_ENTITY_URL)){
+      throw createAndLogErrorRequest(req, `${previousPage} page is not part of the journey!`);
+    }
+    if (req.body["sign_out"] === 'yes') {
+    // This url may change depending on decision made on ticket UAR-267
```

Comment:
I think we can remove this comment. If we were definitely coming back here to change it then we'd not miss it but if we're only maybe coming back, it might end up laying around.

Reply:
This was just a request from Andre so may be best to keep in.

Reply:
I'd be happy to remove now as we know that UAR-267 is not going to be done next/soon.